2.6.md

```diff
@@ -1646,6 +1646,11 @@ This object constitutes a specific deal that was struck between a buyer and a se
     <td>string array</td>  
     <td>Array of advertiser domains (e.g., advertiser.com) allowed to bid on this deal. Omission implies no advertiser restrictions.</td>
    <tr>
+   <tr>
+    <td><code>guar</code></td>
+    <td>integer, default 0</td>  
+    <td>Indicates that the deal is of type ‘guaranteed’ and the bidder must bid on the deal, where 0 = not a guaranteed deal, 1 = guaranteed deal.</td>
```

Comment:
There are cases when guaranteed deals (with a specific volume and price commitment) do not require a bidder to submit a bid for every single bid request where a deal matched – for example, when there's sufficient amount of inventory available to fulfill the volume commitment.

Should this field be literally interpreted as "a bidder must submit a bid", rather than only being an indicator of the deal type?

Reply:
I would suggest that for those cases, the seller should support a deals extension with a 'must_bid' attribute (Google Authorized Buyers does exactly that). While this update doesn't include such an attribute, one could be certainly be proposed.

Reply:
So would the semantics of the proposed `guar` field be solely to indicate the deal type, then, and not whether a bidder must submit a bid? The two seem distinct questions, and it would be great to make the semantics clear and unambiguous.

Reply:
No. As it stands, it signals that the buyer must bid.

Reply:
Is it synonymous then with `must_bid` in Google Authorized Buyers? If so, wonder if we should name it `mustbid`

Reply:
I think that would confuse the issue if it were ever decided to add a separate 'must_bid' signal. As it stands, Google is the only exchange I'm aware of that has the functionality you're suggesting.